lfr/heterogeneous_single_assembly_3D/neutronics_standalone/neutronics.i

```diff
@@ -0,0 +1,1771 @@
+    num_sides = 6 # must be six to use hex pattern
+    num_sectors_per_side = '2 2 2 2 2 2'
+    polygon_size = ${half_pinpitch}
+    ring_radii = '${clad_r_o}'
+    ring_intervals = '1'
+    ring_block_ids = '2'
+    background_intervals = '1'
+    background_block_ids = '3'
+    preserve_volumes = on
+    quad_center_elements = false
+  []
+
+  # This assembles the 7 pins into an assembly with a duct
+  [ASM_CM]
+    type = PatternedHexMeshGenerator
+    inputs = 'PinIn_CM PinOut_CM'
+    pattern_boundary = hexagon
+    background_intervals = '1'
+    hexagon_size = ${half_asmpitch}
+    duct_sizes = '${fparse flat_to_flat/2} ${fparse flat_to_flat/2 + duct_thickness}'
+    duct_sizes_style = apothem
+    duct_intervals = '1 1'
+    background_block_id = '4'
+    duct_block_ids = '5 6'
+    external_boundary_id = 997
+    pattern = '1 1 1 1 1 1 1;
+              1 0 0 0 0 0 0 1;
+             1 0 0 0 0 0 0 0 1;
+            1 0 0 0 0 0 0 0 0 1;
+           1 0 0 0 0 0 0 0 0 0 1;
+          1 0 0 0 0 0 0 0 0 0 0 1;
+         1 0 0 0 0 0 0 0 0 0 0 0 1;
+          1 0 0 0 0 0 0 0 0 0 0 1;
+           1 0 0 0 0 0 0 0 0 0 1;
+            1 0 0 0 0 0 0 0 0 1;
+             1 0 0 0 0 0 0 0 1;
+              1 0 0 0 0 0 0 1;
+               1 1 1 1 1 1 1'
+  []
+  [coarse_mesh]
+    type = AdvancedExtruderGenerator
+    input = ASM_CM
+    heights = '${dhA} ${dhB} ${dhC} ${dhD} ${dhE} ${dhF} ${dhG} ${dhH} ${dhI} ${dhJ}'
+    num_layers = '${num_axmeshA} ${num_axmeshB} ${num_axmeshC} ${num_axmeshD} ${num_axmeshE} ${num_axmeshF} ${num_axmeshG} ${num_axmeshH} ${num_axmeshI} ${num_axmeshJ}'
+    direction = '0 0 1'
+    top_boundary = 998
+    bottom_boundary = 999
+  []
+
+  [cmesh]
+    type = CoarseMeshExtraElementIDGenerator
+    input = assign
+    coarse_mesh = coarse_mesh
+    extra_element_id_name = coarse_element_id
+  []
+  uniform_refine = 0
+[]
+
+# ==============================================================================
+# Transport Systems
+# ==============================================================================
+[TransportSystems]
+  particle = neutron
+  equation_type = eigenvalue
+  G = 9
+  VacuumBoundary = 'ASSEMBLY_TOP ASSEMBLY_BOTTOM'
+  ReflectingBoundary = 'ASSEMBLY_SIDE'
+  [sn]
+    scheme = DFEM-SN
+    family = MONOMIAL
+    order = FIRST
+    AQtype = Gauss-Chebyshev
+    NPolar = 2
+    NAzmthl = 3
+    NA = 1
+    sweep_type = asynchronous_parallel_sweeper
+    using_array_variable = true
+    collapse_scattering  = true
+  []
+[]
+
+# ==============================================================================
+# Executioner
+# ==============================================================================
+[Executioner]
+  type = SweepUpdate
+  verbose = true
+
+  richardson_rel_tol = 1e-4
+  richardson_max_its = 500
+  richardson_value = 'eigenvalue'
+  inner_solve_type = SI
+  max_inner_its = 9
+
+  cmfd_acceleration = true
+  coarse_element_id = coarse_element_id
+  cmfd_eigen_solver_type = krylovshur
```

Comment:
`newton` didn't give a converged solution.

Reply:
Do you know why? I asking because typically we recommend `newton` because it is faster and we possibly can make it work in the future. I am not so concerned using `krylovshur` here.

Reply:
I don't know. In fact, I sent this issue to you and @zachmprince via email to ask for your opinion. This problem could be solved very quickly using the `newton` type and couldn't be converged using the `krylovshur` type about more than 1 year ago. This time, however, only the `krylovshur` type converges the solution, with significantly longer time than the previous `newton` type did in the past.

Reply:
Can you reply that email to bring it to the top on my list? We might just need to change few default values for the newton solver to make it work. I hope CMFD solve is still a small portion of the entire wall time.

Reply:
Oh! Because the email had a zip file attached. INL server filtered it out. The server does send me a notification but I thought it was just a spam... We should share things either with box or with gitlab commits.

Reply:
This is the instruction from the server:
```
If this message is needed, please contact the sender and have them resend the attachment in the allowed format (e.g. filename.inl.zip)
```
@pak1sol can you rename the file and send it again?

Reply:
The result is updated. The computing time is much more reasonable now. I used the Griffin executable built in one of ANL HPC clusters, but I switched to Sawtooth. Not sure about too big difference though.

Reply:
MPI distribution can have quite a big impact. But for now, I think this can be resolved. Looks like Newton option is working.